### PR TITLE
Sprint 114 initial release notes

### PR DESCRIPTION
**Pull request by esbenbach**
Added sprint 114 initial release notes

### CODE REVIEW

File: source/changelog/r114.inc [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+* Support for adding/replacing Payment Request Settings on a Dunning Process via the Dunning Process update endpoint.
+* **[Preview]** New logic for evaluating automatic cancellation due to missing payments (Payment Stop).
+
+Fixed
+-----
+
+* Billing Account transactions would not always apply currency to the datamodel, causing it to be unavailable in the API and reporting/analytics solution.
+* Fixed an issue where removing a future cancellation would lead to invalid accounting and ledger information because any prorated charges would be invoiced again and any allowances would be written off/credited.
+* Fixed an issue where it was impossible to credit an Order Payment Demand for a future subscription that had been deleted.
+* If a billing account had an outstanding Payment Demand with an allowane in exess of the total payment demand amount, and a new payment was registered, the exess allowance would dissapear from the account. This issue has now been fixed.
```
Comment (havmunke) [inline source/changelog/r114.inc:33]: Typo: allowane

File: source/changelog/r114.inc [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+* Fixed an issue where an allowance on the exact Payment Demand amount would cause a reminder to be generated with just a Fee if reminder fees where enabled.
```
Comment (havmunke) [inline source/changelog/r114.inc:34]: This was the no-issue - bug, that was transfered to r115.

File: source/changelog/r114.inc [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+* **[Preview]** Support for Interactive Distribution Innovation Address Lookup (Address Helper) during address registration.
+* Aggregated total of outstanding account transactions from the billing account are now shown in the Subscriber Ledger.
+
+Fixed
+-----
+
+* An issue with the total amount on Subscriber Ledger for Enterprise Plans.
+* An issue where adding a Vipps configuration would present an error message even though the configuration was created.
+* Updating an Organziations main identifier (Organizatoin Number) would reset other identifications configured, this has now been corrected.
```
Comment (havmunke) [inline source/changelog/r114.inc:83]: Typo: Organizatoin

File: source/changelog/r114.inc [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+* An issue where crediting an Invoice for an Enterprise Plan, with a replacement, would lead to an error when generating the replacement Invoice.
+* Contract time was shown incorrectly when changing Subscription Plan.
+* An issue it was not possible to remove a future cancellation.
```
Comment (havmunke) [inline source/changelog/r114.inc:86]: Missing a "where".

File: source/changelog/r114.inc [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+Changed
+-------
+
+* Minor visual improvements to outstanding billing account transactions.
+* Multiple improvements and adjustments to the delivries view.
```
Comment (havmunke) [inline source/changelog/r114.inc:92]: Typo: delivries

File: source/changelog/r114.inc [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+Added
+-----
+
+* **[Preview]** Support for Interactive Distribution Innovation Address Lookup (Address Helper) during address registration.
+
+Fixed
+-----
+
+* When no sales terms are configured and error message will be displayed instead of causing an internal server error.
```
Comment (havmunke) [inline source/changelog/r114.inc:107]: Type: "and" instead of "an"

File: source/changelog/r114.inc [hunk 7]
```diff
@@ -0,0 +1,118 @@
+.. _changelog-r114:
+
+************************
+Release June 13th 2023
+************************
+
+Time flies when you are having fun developing subscription software.
+Its time for another release, the last one before the summer holidays kicks in.
```
Comment (BrynjarS) [inline source/changelog/r114.inc:8]: Typo: its -> It's

**Review by havmunke — APPROVED** [review]

**Pull request merged**